pyaedt/common_rpc.py

```diff
@@ -11,7 +11,9 @@
 import rpyc
 from rpyc.utils.server import ThreadedServer
 from pyaedt.rpc.rpyc_services import GlobalService
+import rpyc.core.consts
 
+rpyc.core.consts.STREAM_CHUNK = 256000000
```

Comment:
Maybe just add a small comment to explain why we must set STREAM_CHUNK to 256000000.

Reply:
Done. Thanks